src/vmm/src/logger/metrics.rs

```diff
@@ -1038,8 +1038,9 @@ pub struct FirecrackerMetrics {
     pub api_server: ApiServerMetrics,
     /// A balloon device's related metrics.
     pub balloon: BalloonDeviceMetrics,
+    #[serde(flatten)]
     /// A block device's related metrics.
-    pub block: block_metrics::BlockDeviceMetrics,
+    pub block_ser: BlockMetricsSerializeProxy,
```

Comment:
why do we need to rename it?

Reply:
The proxy struct is just to get the serialization done within FirecrackerMetrics, it won't be used to actually update any metrics.